lark/parser_frontends.py

```diff
@@ -82,16 +82,18 @@ def __init__(self, lexer_conf, parser_conf, options=None):
         self.postlex = lexer_conf.postlex
 
     @classmethod
-    def deserialize(cls, data, memo, callbacks, postlex, transformer, re_module, debug):
+    def deserialize(cls, data, memo, callbacks, re_module, options):
         inst = super(WithLexer, cls).deserialize(data, memo)
 
-        inst.postlex = postlex
-        inst.parser = LALR_Parser.deserialize(inst.parser, memo, callbacks, debug)
+        inst.postlex = options.postlex
+        inst.parser = LALR_Parser.deserialize(inst.parser, memo, callbacks, options.debug)
 
         terminals = [item for item in memo.values() if isinstance(item, TerminalDef)]
-        inst.lexer_conf.callbacks = _get_lexer_callbacks(transformer, terminals)
+        inst.lexer_conf.callbacks = _get_lexer_callbacks(options.transformer, terminals)
         inst.lexer_conf.re_module = re_module
```

Comment:
Why not fold `re_module` as well?

```python
inst.lexer_conf.re_module = regex if options.regex else re
```

Reply:
It means that we have to import `re` and `regex` into `parser_frontend.py`.

Reply:
I don't know if it makes sense to pass a module as an argument only so we won't have to import it.

Reply:
That is true